src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((stylesObject, style) => {
+      const [key, value] = style.split(':').map((item) => item.trim());
+
+      if (key && value) {
+        stylesObject[key] = value;
+      }
+
+      return stylesObject;
+    }, {});
 }
 
 module.exports = convertToObject;
```

Comment:
The module is correctly exported, allowing the function to be used in other parts of the application or to be tested separately.